packages/wrangler/src/bundle.ts

```diff
@@ -784,9 +792,19 @@ async function applyD1BetaFacade(
 	entry: Entry,
 	tmpDirPath: string,
 	betaD1Shims: string[],
-	local: boolean
+	local: boolean,
+	doBindings: DurableObjectBindings
 ): Promise<Entry> {
 	const targetPath = path.join(tmpDirPath, "d1-beta-facade.entry.js");
+	const maskedDoBindings = doBindings
+		// Don't shim anything not local to this worker
+		.filter((b) => !b.script_name)
+		// Reexport the DO classnames
+		.map(
+			(b) =>
+				`export const ${b.class_name} = maskDurableObjectDefinition(OTHER_EXPORTS.${b.class_name})`
```

Comment:
Don't we want a semicolon after every statement, rather than only in-between multiple export statements?